Float field updates (#6656)
keystonejs/keystone: docs/pages/docs/apis/fields.mdx

> - `defaultValue` (default: `undefined`): Can be either a float value or an async function which takes an argument `({ context, originalInput })` and returns a float value.
>   This value will be used for the field when creating items if no explicit value is set.

Reviewer: ```suggestion
- `defaultValue` (default: `undefined`): Can be a finite float value.
  This value will be used for the field when creating items if no explicit value is set.
```
